Tools/TestWebKitAPI/WebTransportServer.mm

```diff
@@ -40,42 +40,73 @@
 
     Function<Task(Connection)> connectionHandler;
     RetainPtr<nw_listener_t> listener;
-    Vector<RetainPtr<nw_connection_group_t>> connectionGroups;
+    RetainPtr<nw_connection_group_t> connectionGroup;
     Vector<RetainPtr<nw_connection_t>> connections;
     Vector<CoroutineHandle<Task::promise_type>> coroutineHandles;
 };
 
 WebTransportServer::WebTransportServer(Function<Task(Connection)>&& connectionHandler)
     : m_data(Data::create(WTFMove(connectionHandler)))
 {
+    RetainPtr<nw_parameters_t> parameters = nullptr;
+    RetainPtr<nw_protocol_options_t> webtransportOptions = nullptr;
+
+#if HAVE(WEB_TRANSPORT)
+    auto configureWebTransport = [](nw_protocol_options_t options) {
+        nw_webtransport_options_set_is_datagram(options, true);
+        nw_webtransport_options_set_is_unidirectional(options, false);
+        nw_webtransport_options_set_connection_max_sessions(options, 1);
+    };
+
     auto configureTLS = [](nw_protocol_options_t options) {
-        RetainPtr securityOptions = adoptNS(nw_quic_connection_copy_sec_protocol_options(options));
+        RetainPtr securityOptions = adoptNS(nw_tls_copy_sec_protocol_options(options));
         sec_protocol_options_set_local_identity(securityOptions.get(), adoptNS(sec_identity_create(testIdentity().get())).get());
-        sec_protocol_options_add_tls_application_protocol(securityOptions.get(), "h3");
     };
 
-    RetainPtr parameters = adoptNS(nw_parameters_create_quic_stream(NW_PARAMETERS_DEFAULT_CONFIGURATION, configureTLS));
+    auto configureQUIC = [](nw_protocol_options_t options) {
+        nw_quic_set_initial_max_streams_bidirectional(options, 1000);
```

Comment:
Do we need this?  Can we give 1000 a name?  Can we use std::numeric_limits like we did in the client?